applications/visit-stampede3/app.json

```diff
@@ -87,7 +87,7 @@
       "portalName: DesignSafe"
     ],
     "notes": {
-      "icon": "Generic-Vis",
+      "icon": "Generic-Visualization",
```

Comment:
This is intentional change?

Reply:
Yes, it matches what's in the drop down menu in the App Listing Entry